starter-project-web-vue2/pages/andualem/index.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <v-app>
+    <v-dialog max-width="600px">
+        <template v-slot:activator="{ on }">
```

Comment:
Its recommended you use components in pages. copy different parts of the page into components as needed and only refer the components here and pass the data as props to the components

Reply:
contact me if you need help with that t.me/ebisu0